src/input.rs

```diff
@@ -153,6 +162,7 @@ impl<'a> Input<'a> {
                         return self.handle_submit();
                     }
                 }
+                Key::Tab => self.handle_tab()?,
```

Comment:
In case of `huh`, the key `Ctrl+E (End)` is used since `Tab` is used to switch between the fields within a form. I find this a bit counter intuitive but if we choose to implement forms/groups (which seems non-trivial with that nasty cursor handling) we might also need to go that route.